packages/charts/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -24,9 +24,10 @@ export interface IDonutChartState {
   xCalloutValue?: string;
   yCalloutValue?: string;
   focusedArcId?: string;
-  selectedLegend: string;
+  selectedLegend?: string;
```

Comment:
>selectedLegend [](http://example.com/codeflow?start=2&length=14)

Selected legend is not being used anywhere

Reply:
removed